operator/internal/manifests/internal/config/build_test.go

```diff
@@ -5418,3 +5418,296 @@ analytics:
 		})
 	}
 }
+
+func TestBuild_ConfigAndRuntimeConfig_AddSchema(t *testing.T) {
```

Comment:
I'm not sure what we're testing here. Do we still need this test or is it a left-over from previous versions of this PR?

Reply:
It's a leftover from the previous version of the PR